src/types/pivot.ts

```diff
@@ -17,9 +17,7 @@ export type Aggregator =
 
 export type Granularity =
   | "day"
-  | "week"
```

Comment:
Could be worth to make it in another commit, to explain why. I remember we discussed it, but in two weeks I will not remember anymore :D

Reply:
I discussed it with @pro-odoo , the commit message could be more precise, if im' not mistaken, those granularities are not used for spreadsheet pivots specifically